src/components/app/index.ts

```diff
@@ -63,9 +68,9 @@ export const create = (
   // kosko component env values
   const envParams = merge(
     defaultEnvParams, // set name as default if not provided
-    gitlab(process.env),
+    config ?? {}, // create options
     env.component(name) as AppConfig, // kosko env overrides
-    config ?? {} // create options
+    gitlab(process.env),
```

Comment:
this makes it impossible to override what gitlab returns :/ (ex: subdomain)

Reply:
should be fixed in https://github.com/SocialGouv/kosko-charts/pull/317